src/nonfeatures/README.md

```diff
@@ -2,14 +2,18 @@
 
 Certain FreeBSD features are currently unavailable in CheriBSD, including:
 
-- ZFS
 - DTrace
 
 Many other kernel features are not yet well validated, including:
 
 - Non-UFS file systems
 - Most optional modules, such as various firewall systems
 
+The 22.12 release includes Alpha support for the ZFS file system.  It is
```

Comment:
What about adding a subsection on "ZFS Alpha support", and a list entry ZFS in the list "Many other kernel features are not yet well validated"? It would be better visible then.